docs/src/main/sphinx/security/ldap.rst

```diff
@@ -55,10 +55,11 @@ Property                                                      Description
                                                               Should be set to ``true``. Default value is
                                                               ``false``.
 ``http-server.https.port``                                    HTTPS server port.
-``http-server.https.keystore.path``                           The location of the Java Keystore file that will be
-                                                              used to secure TLS.
-``http-server.https.keystore.key``                            The password for the keystore. This must match the
-                                                              password you specified when creating the keystore.
+``http-server.https.keystore.path``                           The location of the PEM or Java keystore that
+                                                              is used to secure TLS.
+``http-server.https.keystore.key``                            The password for the PEM or Java keystore. This
+                                                              must match the password you specified when creating
+                                                              the PEM or keystore.
```

Comment:
the PEM or JKS keystore

Reply:
Dain made me take out all my original "JKS keystone" phrases in favor of just "keystone".

Reply:
hm.. but ok.